src/main/kotlin/no/nav/arrangor/domain/Ansatt.kt

```diff
@@ -4,6 +4,7 @@ import java.util.UUID
 
 data class Ansatt(
 	val id: UUID,
+	val source: String? = "amt-arrangor",
```

Comment:
Det blir feil å sette defaultverdi for denne og for source på ArrangorDto til amt-arrangør. Hvis disse er null er jo sourcen amt-tiltak, siden den har publisert uten source lenge :) Jeg tror det tryggeste er å la være å sette defaultverdier for disse.